src/main/java/com/robomwm/prettysimpleshop/feature/ShowoffItem.java

```diff
@@ -8,10 +8,7 @@
 import com.robomwm.prettysimpleshop.event.ShopSelectEvent;
 import com.robomwm.prettysimpleshop.shop.ShopAPI;
 import com.robomwm.prettysimpleshop.shop.ShopInfo;
-import org.bukkit.Chunk;
-import org.bukkit.ChunkSnapshot;
-import org.bukkit.Location;
-import org.bukkit.World;
+import org.bukkit.*;
```

Comment:
Any possibility you can tell your IDE not to do this? Otherwise I can just fix it before I merge then.

Reply:
there done